src/backend/functions/change-requests.ts

```diff
@@ -110,12 +111,102 @@ const getChangeRequestByID: ApiRouteFunction = async (params: { id: string }) =>
   return buildSuccessResponse(changeRequestTransformer(requestedCR));
 };
 
+const createNewChangeRequest: ApiRouteFunction = async (_params, event) => {
+  if (event.body === undefined) {
+    return buildClientFailureResponse('No data for CR creation.');
+  }
+  const body = JSON.parse(event.body!);
+  if (
+    body.submitterId === undefined ||
+    body.wbsElementId === undefined ||
+    body.type === undefined
+  ) {
+    return buildClientFailureResponse('Missing common CR field(s).');
+  }
+  if (!(body.submitterId >= 0) || !(body.wbsElementId >= 0)) {
+    return buildClientFailureResponse('ID field(s) must be integers.');
+  }
+  if (
+    body.type === CR_Type.ISSUE ||
+    body.type === CR_Type.DEFINITION_CHANGE ||
+    body.type === CR_Type.OTHER
+  ) {
+    if (
+      body.what === undefined ||
+      body.scopeImpact === undefined ||
+      body.timelineImpact === undefined ||
+      body.budgetImpact === undefined ||
+      body.why === undefined
+    ) {
+      return buildClientFailureResponse('Missing standard CR field(s).');
+    }
+    console.log(body.timelineImpact >= 0);
```

Comment:
I can't speak for the rest of the code base, but I'm wondering about logging this "timeline impact." I'm wondering about logging the CR type that was detected when this function was called. Is there documentation somewhere on what we're logging in general and why?

Reply:
This log line is totally meaningless. I logged stuff to see what was going on / debug. I haven't even begun to think about actual purposeful logging.